srcpkgs/ecl/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'ecl'
+pkgname=ecl
+version=16.1.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="m4"
```

Comment:
add this line directly below hostmakedepends=

```
depends="gc-devel libatomic_ops-devel gmp-devel"
```

Reply:
I think this is `makedepends=...` yes?

Reply:
yes, makedepends=, sorry it was late